arrow-libs/core/arrow-core-test/src/main/kotlin/arrow/core/test/laws/MonoidLaws.kt

```diff
@@ -1,37 +1,36 @@
 package arrow.core.test.laws
 
-import arrow.typeclasses.Eq
 import arrow.typeclasses.Monoid
 import io.kotlintest.properties.Gen
 import io.kotlintest.properties.forAll
 import io.kotlintest.shouldBe
 
 object MonoidLaws {
 
-  fun <F> laws(M: Monoid<F>, GEN: Gen<F>, EQ: Eq<F>): List<Law> =
-    SemigroupLaws.laws(M, GEN, EQ) +
+  fun <F> laws(M: Monoid<F>, GEN: Gen<F>, f: (F, F) -> Boolean = { a, b -> a?.equals(b) == true }): List<Law> =
```

Comment:
this default value could be a `Eq.default()` that can be used in multiple places if you go for the typealias

Reply:
`Eq.default` or `Eq.any` like we had before can still be used like this but be replaced with a simple lambda.